mdsuite/transformations/transformation_dict.py

```diff
@@ -84,13 +84,15 @@
     mdp.ionic_current: ionic_current.IonicCurrent,
     mdp.kinaci_heat_current: kinaci_integrated_heat_current.KinaciIntegratedHeatCurrent,
     mdp.momentum_flux: momentum_flux.MomentumFlux,
-    mdp.scaled_positions: scale_coordinates.ScaleCoordinates,
+    mdp.positions: [
+        scale_coordinates.ScaleCoordinates,
+        wrap_coordinates.CoordinateWrapper,
```

Comment:
I'm not sure about this whole file but I guess it is necessary.

One thing that we might want to change would be 
`from mdsuite.transformations import unwrap_coordinates; unwrap_coordinates.CoordinateUnwrapper` 
to add all of them to the `mdsuite.transformations.__init__` to the `__all__` and then simply have `from mdsuite.transformations import CoordinateUnwrapper` ?

Reply:
sadly I have not found a way to circumvent this, I know it is bad design.
I am not a fan of the __all__ mechanism (bcs it uses strings to point to classes which I hate).
Since the transformations are very small now anyway, we could just put them all in transformations.py to make importing easier

Reply:
Have you tried `__all__ = [CoordinateUnwrapper.__name__, ...]` ?

Reply:
nope. but that sounds much better already